build.sbt

```diff
@@ -30,7 +30,7 @@ versionScheme := Some("semver-spec")
 libraryDependencies ++= Seq(
   "com.typesafe.play" %% "play" % playVersion % "provided",
   "org.webjars" % "requirejs" % "2.3.6",
-  "org.webjars" % "webjars-locator" % "0.46" exclude("com.fasterxml.jackson.core", "jackson-databind"),
+  "org.webjars" % "webjars-locator" % "0.47" exclude("com.fasterxml.jackson.core", "jackson-databind"),
```

Comment:
```suggestion
  "org.webjars" % "webjars-locator" % "0.50" exclude("com.fasterxml.jackson.core", "jackson-databind"),
```

Reply:
Not sure if we still need the `exclude`

Reply:
hmm.... yeah the exclude was introduced in cf2ff330c011b4e7ac6b8bdba96d5de15b924126 I guess this was because of what I said about conflicts. We can try to remove it and see what happens. Should be safe now if we take care which jackson versions we pull in via webjars-locator